src/components/Logs/Table/logs-table.js

```diff
@@ -39,13 +94,27 @@ export function LogsTable({ logs, actions = [] }) {
               <button
                 key={action.label}
                 type="button"
-                disabled={!(selectedList.length > 0)}
-                onClick={() => action.handler(selectedList)}
-                className={`${
-                  selectedList.length > 0
+                disabled={!Object.keys(selectedRowIds).length}
+                onClick={() =>
+                  action.handler(
```

Comment:
action.handler раньше принимала массив выбранных логов и сейчас я не вижу в ней изменений. так что думаю правильней будет так

```suggestion
                  action.handler(selectedFlatRows.map(
                          (d) => d.original
                        ))
```

но заметь, что библиотека добавляет к итемам в этом массиве какое-то доп поле. наверное его лучше убрать